docs/component-guides/notebooks.md

```diff
@@ -0,0 +1,71 @@
+## (Optional) Setup RDS & S3 credentials access for notebooks & pipelines
+### Use Cases
+- Be able to use `boto3` or aws libraries that require to pass in credentials in a notebook and specify credentials without hard coding them. Access the credentials via environment variables.  
+- Be able to use [ml-metadata](https://github.com/google/ml-metadata/blob/master/g3doc/get_started.md) in a notebook (eg: to explore metadata) and specify the credentials needed by using environment variables.
+- Be able to use [ml-metadata](https://github.com/google/ml-metadata/blob/master/g3doc/get_started.md) to query metadata during a pipeline run by passing a kubernetes secret to a pipeline component.  
+- Be able to use `boto3` or aws libraries that require to pass in credentials in a Kubeflow Pipeline Component. 
+
+The following script creates a kubernetes `mysql-secret` and `mlpipeline-minio-artifact` secrets with RDS & S3 credentials specified in AWS Secrets Manager created while deploying the platform. This is a sample for demonstrating how you can use [`PodDefault` resource](https://github.com/kubeflow/kubeflow/blob/master/components/admission-webhook/README.md) and secrets in Notebooks to access the metadata database and and artifacts in S3 bucket created by pipelines. Make sure you create separate database and IAM users and corresponding secrets in Secrets Manager for your users if you want fine grain access control and auditing.  
+
+### Setup the secrets access
+1. Navigate inside the script directory.
+```
+cd ../../tests/e2e
+```
+2. Install the script dependencies.
+```shell
+pip install -r requirements.txt
+```
+3. Execute the script, replace `YOUR_CLUSTER_REGION`, `YOUR_CLUSTER_NAME` and `YOUR_NAMESPACE` with the appropriate values.  
+Note that `YOUR_NAMESPACE` represents the namespace the secrets will be setup for.  
+For instance if your notebooks and pipelines will be in `kubeflow-user-example-com`, then you would use `kubeflow-user-example-com` in place of `YOUR_NAMESPACE`.
+```shell
+PYTHONPATH=.. python utils/notebooks/setup_secrets_access.py --region YOUR_CLUSTER_REGION --cluster YOUR_CLUSTER_NAME --profile-namespace YOUR_NAMESPACE
+```  
+You can always do `PYTHONPATH=.. python utils/notebooks/setup_secrets_access.py --help` to learn more about the parameters available.    
+
+**Note :** The namespace must exist before executing the script.   
+
+### (Optional) Set PodDefault selected by default in notebook UI  
```

Comment:
https://github.com/awslabs/kubeflow-manifests/pull/106#discussion_r822157387

Can you help create a PR to the main Kubeflow docs to address this? I agree the upstream docs only talk about how to make some configurations default

Reply:
I just realized that we should stick to having this configuration optional same as the Kubeflow docs as this change will introduce a dependency for this `PodDefault` to be available in all profile namespaces. Existing profile namespaces which do not have this resource will see failure in new notebook server creation and same for new namespaces

Reply:
@surajkota We can make a PR to update the kubeflow doc but it might be less intuitive for the user if we remove it from aws doc.  
The kubeflow doc will not be ableto give precise instructions on the value to add for aws since the pod default label would be different on GCP vs AWS for instance.  
Maybe we can add it to both, so have a more generic doc in Kubeflow doc, then to keep the current doc with the preicse label name so the user doesn't have to know too much about k8s or doesn't have to scan the pod default file to find the label that is applied in the script to get it setup.  
  
What do you think?

Reply:
Okay, for ease of use lets have this but add it to the doc:

Making this configuration default will introduce a dependency for the secrets and PodDefault to be available in all profile namespaces. Existing profile namespaces which do not have this resource will see failure in new notebook server creation if this PodDefault is not created

Reply:
Done